db/migrate/20180109003825_add_reply_order_to_replies.rb

```diff
@@ -0,0 +1,19 @@
+class AddReplyOrderToReplies < ActiveRecord::Migration[5.1]
```

Comment:
I swear, every time we say we'll Do It Differently and every time we don't, but I really need to start, so. Migrations first, and separate, please. I want to get out of the habit of having to put the site into maintenance mode every time we release something with a db change.

Reply:
(Moved out to #688)